docs/dosbox.1

```diff
@@ -162,7 +161,7 @@ Sets the amount of free space available on a drive in MB's. This is a more
 .RI "Sets the name of the drive to " drivelabel ". Needed on some" 
 systems if the cd label isn't read correctly. Useful when a 
 program can't find its cdrom. If you don't specify a label and no
-.RB "lowlevel support is selected (" "\-usecd #" " and/or " "\-ioctl" "):"
+.RB "lowlevel support is selected (" "\-usecd #" "):"
```

Comment:
I think `-ioctl` is still valid, because that's using the SDL interface (if I understand correctly?)